tests/integration-test.sh

```diff
@@ -380,6 +380,8 @@ log "Waiting for node 1 to send messages to node 5"
 for j in ${jobs[@]}; do wait -n $j; done; jobs=()
 log "Waiting DONE"
 
+api_aggregate_tickets "${api2}" "0x8b4f8683b34e36012804062d6a6dbb2b1f814876a59fe566d4badf0b011db756" 
```

Comment:
This test is insufficient. We should open a new channel. Generate tickets by sending packets through it. Verify it has `x` tickets. Run aggregation, then verify it has only 1 ticket, then redeem that 1 ticket.

Reply:
@tolbrino is that a test you were working on?

Reply:
yes